metricbeat/docs/modules/elasticsearch.asciidoc

```diff
@@ -13,7 +13,15 @@ The `elasticsearch` module collects metrics about {es}.
 The `elasticsearch` module works with {es} 6.7.0 and later.
 
 [float]
-=== Usage for Stack Monitoring
+=== Module-specific configuration notes
+
+Like other {beatname_uc} modules, the `elasticsearch` module accepts a `hosts` configuration setting.
+This setting can contain a list of entries. The related `scope` setting determines how each entry in
+the `hosts` list is interpreted by the module.
```

Comment:
```suggestion
the `hosts` list is interpreted by the module.

```

Reply:
Sorry, noticed one more issue. You need a blank line or the list does not render as a bulleted list.